examples/README.md

```diff
@@ -0,0 +1,14 @@
+# Examples
+This directory contains examples on how to build a firecrown likelihood that could be used to to blind datavectors. 
+
+These need to contain the following methods:
+- `build_likelihood` which returns a `firecrown.Likelihood` object
+- `.compute_theory_vector(ModellingTools)` which returns the theory vector for the given model
+- `.get_data_vector()` which returns the data vector from the likelihood
+
```

Comment:
In principle this shouldn't be necessary if you're already assuming a firecrown likelihood. At least once these methods have been defined in a base class.